README.md

```diff
@@ -251,9 +243,15 @@ type Receipt struct {
   # delegation chain from executor to the issuer. This should be 
   # omitted when the executor is the issuer.
   prf     [&UCAN]
+}
 
-  # Signature from the "iss".
-  s       Varsig
+type Receipt struct {
+  ocm     Outcome
+  sig     Varsig
+}
```

Comment:
This comment...

https://github.com/ChainAgnostic/varsig/pull/6#discussion_r1151553722

...mentions this...

<img width="743" alt="Screenshot 2023-03-29 at 23 23 18" src="https://user-images.githubusercontent.com/1052016/228747697-2747835d-06c7-4fea-84ec-fec836c54ca0.png">

...which is also my assumption about how varsig will work. If that gets merged, we could consider not having a `Receipt` wrapper here at all, since it would be redundant.